problem_details.py

```diff
@@ -2,32 +2,42 @@
 from util.http import INTEGRATION_ERROR
 from flask.ext.babel import lazy_gettext as _
 
+# Generic problem detail documents that recapitulate HTTP errors.
+# call detailed() to add more specific information.
+
```

Comment:
I needed these for a number of cases in circ manager. Since they are generic versions of HTTP errors I put them in core.